pyface/i_widget.py

```diff
@@ -144,12 +147,16 @@ def destroy(self):
     def _create(self):
         """ Creates the toolkit specific control.
 
-        This method should create the control and assign it to the
-        :py:attr:``control`` trait.
+        The default implementation simply calls create()
         """
-        self.control = self._create_control(self.parent)
-        self._initialize_control()
-        self._add_event_listeners()
+        from warnings import warn
+
+        warn(
+            "The _create() method will be removed in a future version of "
+            "Pyface.  Use create() instead.",
+            PendingDeprecationWarning,
```

Comment:
OT: I'm still not sure I understand the use-cases for `PendingDeprecationWarning`; it's tempting just to avoid it and use `DeprecationWarning` everywhere instead.

Reply:
In my mind the difference is between "this is going to go away; but we're still using it internally" vs "this is going to go away; and we aren't using it internally any more".  That allows two levels of warning for downstream projects.

But yes, this should probably be a deprecation warning.